src/Microsoft.DotNet.ProjectJsonMigration/transforms/IncludeContextTransform.cs

```diff
@@ -36,6 +46,11 @@ internal class IncludeContextTransform : ConditionalTransform<IncludeContext, IE
                         fullIncludeSet = fullIncludeSet.Union(includeContext.BuiltInsInclude.OrEmptyIfNull());
                     }
 
+                    if (_patternsBlackList != null)
```

Comment:
You are double checking this. Once here and once inside the IsPatternBlacklisted. I still prefer you assign an empty list if null is passed in the constructor and don't do these null checks anywhere.